packages/diagrams/src/index.tsx

```diff
@@ -3,8 +3,6 @@ import { useEffect, useId, useState } from 'react';
 import classNames from 'classnames';
 import type { Mermaid } from 'mermaid';
 
-// Follow previous flow of importing mermaid dynamically.
```

Comment:
Ah, I meant to leave that in! I'm sure someone will still find this thread if needs be.